application/lib/ui/start_screen_ui.dart

```diff
@@ -22,21 +22,23 @@ class _StartScreen extends State<StartScreen> {
       backgroundColor: MyColors.backgroundColor,
       body: Center(
         child: Container(
-          padding: const EdgeInsets.all(10),
-          margin:
-              const EdgeInsets.only(left: 20, top: 250, right: 20, bottom: 90),
+          margin: const EdgeInsets.only(left: 20, top: 180, right: 20, bottom: 90),
```

Comment:
A margint csak arra használjuk, hogy térközt állítsunk be két elem között vagy egy minimális eltartást tegyünk az eszköz széle és az elem közé. Az elemek pozícionálására nem jó, mert hiába írjuk át mindig másik beégetett számra az értéket, jöhet olyan képernyő, ahol szintén nem fog működni. Ezért szoktuk pl CSS-ben százalékos arányokkal megadni, hogy melyik elem hány százalékát töltse ki a képernyőnek. Flutterben ilyen százalékos lehetőség nincs, viszont van hányados lehetőség, ami pl azt jelenti, hogyha van 3 darab elemed, azoknak a flex értéke 1-1-1 akkor mindegyik elem a képernyő 1/3-át fogja elfoglalni. Ha a flex értékeke 3-2-1, akkor az első elem a képernyő 3/6-át a második elem a 2/6át a harmadik elem pedig az 1/6át.  Az expanded widget a flex értékkel kombinálva például egy tökéletes választás ahhoz hogy responsive képernyőket készítsünk. 
Itt egy cikk is róla: 
https://farizmamad.medium.com/code-the-flex-using-dart-flutter-ae07b2831237